packages/SketchMorph2-Core.package/M2Backend.class/instance/drawEraser.st

```diff
@@ -5,4 +5,5 @@ drawEraser
 		isErasing: true;
 		mode: #pen;
 		currentTool: #drawEraser;
+		currentCursor: (Form dotOfSize: self eraserSize color: Color white);
```

Comment:
I don't really like the concept of a white cursor on a mostly white background. When there is not much color on the canvas, one can barely find the cursor.